pkg/exporter/sumologicexporter/config.go

```diff
@@ -198,11 +200,13 @@ const (
 	// OTLPTraceFormat represents trace_format: otlp
 	OTLPTraceFormat TraceFormatType = "otlp"
 	// GZIPCompression represents compress_encoding: gzip
-	GZIPCompression CompressEncodingType = "gzip"
+	GZIPCompression configcompression.CompressionType = "gzip"
+	// ZSTDCompression represents compress_encoding: zstd
+	ZSTDCompression configcompression.CompressionType = "zstd"
 	// DeflateCompression represents compress_encoding: deflate
-	DeflateCompression CompressEncodingType = "deflate"
+	DeflateCompression configcompression.CompressionType = "deflate"
 	// NoCompression represents disabled compression
-	NoCompression CompressEncodingType = ""
+	NoCompression configcompression.CompressionType = ""
```

Comment:
Is there any reason to define these constants vs just using the ones from the core package?

Reply:
ah yes, I'll just use the upstream ones.